examples/datasets/load_limo_dataset.py

```diff
@@ -0,0 +1,173 @@
+"""
+========================================================
+Import single subject from LIMO data set into MNE-Python
+========================================================
+
+Here we a define function to extract the eeg signal data
+from the LIMO structures in the LIMO dataset, see [1]_ and:
+
+    https://datashare.is.ed.ac.uk/handle/10283/2189?show=full
+
+    https://github.com/LIMO-EEG-Toolbox
+
+The code allows to:
+
+    - Fetch single subjects epochs data for the LIMO data set.
+    - Get the epochs information from the LIMO .mat files, such as
+      sampling rate, number of epochs per condition, number and name
+      of EEG channels per subject.
+    - Create a custom info and epochs structure in MNE-Python.
+    - Interpolate missing channels.
+    - Plot the evoked response for experimental conditions.
+
+.. note:: This example assumes that the LIMO data set has already
+          been downloaded and stored in home directory.
+
+
+References
+----------
+.. [1] Guillaume, Rousselet. (2016). LIMO EEG Dataset, [dataset].
+       University of Edinburgh, Centre for Clinical Brain Sciences.
+       https://doi.org/10.7488/ds/1556.
+"""
+
+# Authors: Jose C. Garcia Alanis <alanis.jcg@gmail.com>
+#
+#
+# License: BSD (3-clause)
+
+
+# import necessary packages
+import os.path as op
+import scipy.io
+import numpy as np
+import mne
+
+print(__doc__)
+
+
+# --- define load limo epochs function ---
+
+
+def load_limo_epochs(subject, path=None, interpolate=False):
+    """Fetch subjects epochs data for the LIMO data set.
+    Parameters
+    ----------
+    path : str
+        The path to store the downloaded data. Defaults to home directory.
+    subject : int | str
+        Subject to use. Can be in the range from 2 to 18.
+        If string, must be 'S1', 'S2' etc.
```

Comment:
Actually, do we really want to support both str and int? Did we discuss this?

Reply:
No, I don't think so. I left it as it is for now. But I can change the input parameters if you guys think it's better.